### PR TITLE
Add data plane manifests for KafkaSource

### PR DESCRIPTION
**Pull request by pierDipi**
Part of #312

## Proposed Changes

- Add data plane manifests for KafkaSource

### CODE REVIEW

File: data-plane/config/source/200-source-data-plane-cluster-role.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+
+# Copyright 2020 The Knative Authors
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply data-plane/config/source/200-source-data-plane-cluster-role.yaml]: Done.

File: control-plane/cmd/kafka-controller/main.go [hunk 2]
```diff
@@ -63,5 +69,10 @@ func main() {
 		func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
 			return sink.NewController(ctx, watcher, sinkEnv)
 		},
+
+		// KafkaSource controller
+		func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+			return source.NewController(ctx, watcher, sourceEnv)
+		},
```
Comment (pierDipi) [inline control-plane/cmd/kafka-controller/main.go:76]: I have to add this since otherwise the data plane won't start

Comment (matzew) [reply control-plane/cmd/kafka-controller/main.go:76]: funcy ....

**Pull request closed without merging**